dashboard/test/ui/features/teacher_tools/script_overview.feature

```diff
@@ -61,6 +61,7 @@ Feature: Unit overview page
     # On last level of the lesson
     And I am on "http://studio.code.org/s/csp3-2019/lessons/3/levels/1"
     And I click selector ".submitButton"
+    And I wait until I am on "http://studio.code.org/s/csp3-2019"
```

Comment:
Curious about the difference between this line and the line below. Excited to see if it helps!